docs/product/explore/profiling/getting-started.mdx

```diff
@@ -4,39 +4,54 @@ sidebar_order: 2
 description: "Get started with Profiling, which allows you to see code-level profiling information for your Sentry apps."
 ---
 
-To set up the profiling integration, use the links for supported SDKs below. Profiling helps you see code-level information about your apps monitored by Sentry.
+Continuous Profiling and UI Profiling are the latest iteration of Sentry’s profiling capabilities, but they are currently only supported in select SDKs as described below. 
+
+Other platforms are supported via the prior transaction-based Profiling product, but these will not benefit from new capabilities introduced by Continuous and UI Profiling (direct start/stop control over the profile lifecycle and removal of duration limits). For more information on the differences between transaction-based Profiling and Continuous/UI Profiling, read [this documentation](/product/explore/profiling/transaction-vs-continuous-profiling).
+
+If you are currently using transaction-based Profiling and want to migrate to Continuous Profiling or UI Profiling on a supported SDK, read the [migration guide](/product/explore/profiling/continuous-ui-profiling-migration-guide).
+
+All SDKs that currently support transaction-based Profiling will be migrated over time to support Continuous Profiling and UI Profiling.
 
 ## Supported SDKs
 
+### Transaction-based Profiling
+
 <Alert>
 
-Profiling depends on Sentry's performance monitoring product being enabled beforehand. To enable performance monitoring in the SDK, check out our [Insights guide](/product/insights/).
+Transaction-based Profiling depends on Sentry's tracing product being enabled beforehand. To enable tracing and performance monitoring features in the SDK, check out our [Insights guide](/product/insights/).
 
 </Alert>
 
-### Transaction Profiling
-
 - [Browser JavaScript](/platforms/javascript/profiling/) [beta]
 - Mobile
-  - [Android](/platforms/android/profiling/)
-  - [iOS and macOS](/platforms/apple/guides/ios/profiling/)
   - [React Native](/platforms/react-native/profiling/) [beta]
   - [Flutter](/platforms/dart/guides/flutter/profiling/) [experimental, iOS and macOS only]
   - [.NET](/platforms/dotnet/guides/maui/profiling/) [experimental, iOS only]
 - Standalone and server apps
-  - [Node.js](/platforms/javascript/guides/node/profiling/)
-  - [Python](/platforms/python/profiling/)
   - [PHP](/platforms/php/profiling/)
   - [Ruby](/platforms/ruby/profiling/) [beta]
   - [.NET](/platforms/dotnet/profiling/) [experimental]
 
 ### Continuous Profiling
```

Comment:
@indragiek - you would know better than me in this case; but since we want to encourage the usage of the new capabilities here - should we make continuous / ui  profiling the 1st and 2nd, and place transaction below it with a note reinforcing "If your SDK isn't supported; transaction based is supported on these SDKs"? 

Thought is to draw focus into the new functionality ones.